eks_infra/env_setup.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#/ Usage: ./env_setup.sh
+
+set -ex
+
+function install_kubectl_client() {
+    curl --silent --location https://amazon-eks.s3-us-west-2.amazonaws.com/1.20.4/2021-04-12/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl
+    chmod +x /usr/local/bin/kubectl
+}
+# aws caller identity
+aws sts get-caller-identity
+
+# install aws-iam-authenticator
+if ! [ -x "$(command -v aws-iam-authenticator)" ]; then
+    curl --silent --location https://amazon-eks.s3-us-west-2.amazonaws.com/1.20.4/2021-04-12/bin/linux/amd64/aws-iam-authenticator -o /usr/local/bin/aws-iam-authenticator
```

Comment:
can we get this `1.20.4` from the json file? if I update the params, for instance, I still need to update these values separately

Reply:
same here.